pkgs/development/python-modules/gios/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, aiohttp
+, aioresponses
+, buildPythonPackage
+, fetchFromGitHub
+, pytest-asyncio
+, pytest-error-for-skips
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "gios";
+  version = "1.0.1";
+  disabled = pythonOlder "3.7";
```

Comment:
does this even work?

```

self = <homeassistant.components.gios.config_flow.GiosFlowHandler object at 0x7ffdc9f5c4f0>
user_input = {'name': 'Foo', 'station_id': 123}

    async def async_step_user(self, user_input=None):
        """Handle a flow initialized by the user."""
        errors = {}

        if user_input is not None:
            try:
                await self.async_set_unique_id(
                    user_input[CONF_STATION_ID], raise_on_progress=False
                )
                self._abort_if_unique_id_configured()

                websession = async_get_clientsession(self.hass)

                with timeout(30):
                    gios = Gios(user_input[CONF_STATION_ID], websession)
>                   await gios.update()
E                   AttributeError: 'Gios' object has no attribute 'update'
```

Got quite a few other errors

Reply:
The tests for Home Assistant will start to work when Home Assistant is updated to 2021.6.0. Thus, this PR is marked as "on hold".